testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1674,6 +1674,36 @@ public void testRenameTableAcrossSchema()
         assertFalse(getQueryRunner().tableExists(getSession(), renamedTable));
     }
 
+    @Test
+    public void testRenameTableUnqualifiedName()
+    {
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_SCHEMA) && hasBehavior(SUPPORTS_CREATE_TABLE));
+
+        String sourceSchemaName = "test_source_schema_" + randomTableSuffix();
+        assertUpdate("CREATE SCHEMA " + sourceSchemaName);
+
+        String tableName = "test_rename_unqualified_name_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + sourceSchemaName + "." + tableName + " AS SELECT 123 x", 1);
+
+        String renamedTable = "test_rename_unqualified_name_new_" + randomTableSuffix();
+        if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
```

Comment:
i'd move this to `skipTestUnless` part

Reply:
You are right. There is already a specific test for renaming tables so there's no need to do the check here again.